changelog.d/19643_only_set_connection_close_http1.fix.md

```diff
@@ -0,0 +1,3 @@
+When terminating idle HTTP connections using the configured `max_connection_age`, only send
+`Connection: Close` for HTTP/0.9, HTTP/1.0, and HTTP/1.1 requests. This header is not supported on
+HTTP/2 and HTTP/3 requests. This may be supported on these HTTP versions in the future.
```

Comment:
👀  did the generate script not pull this one in? 😟

Reply:
Ah, v0.35.0 was before the tooling for generating changelogs existed so I manually created this one.